### PR TITLE
output: Handle Fluent Bit V2 metadata format

### PR DESCRIPTION
**Pull request by cosmo0920**
Closes https://github.com/fluent/fluent-bit-go/issues/66

### CODE REVIEW

**Review by l2dy — COMMENTED** [review]
Could you write a test case for this? It's getting complicated.

**Comment by cosmo0920** [comment]
@l2dy I added a test case for confirming how decoder works for Fluent Bit V2 format.

File: output/decoder.go [hunk 1]
```diff
@@ -82,7 +82,22 @@ func GetRecord(dec *FLBDecoder) (ret int, ts interface{}, rec map[interface{}]in
...
+	case uint64:
+		t = ty
+	case []interface{}: // for Fluent Bit V2 metadata type of format
+		s := reflect.ValueOf(ty)
+		if s.Kind() != reflect.Slice || s.Len() < 2 {
+			return -4, 0, nil
+		}
+		t = s.Index(0).Interface()
+	default:
+		return -5, 0, nil
```
Comment (l2dy) [inline output/decoder.go:99]: Are `FLBTime` and `uint64` the only possible types in Fluent Bit V1?

**Comment by cosmo0920** [comment]
@edsiper @niedbalski Could you approve this change? I need your +1 power.

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**